Source/SPSSHTunnel.m

```diff
@@ -366,7 +366,7 @@ - (void)launchTask:(id) dummy
 		TA(@"-o", [NSString stringWithFormat:@"UserKnownHostsFile=%@/.keys/ssh_known_hosts_strict", NSHomeDirectory()]);
 		
 		// Use a custom ssh config file
-		TA(@"-F", [[NSBundle mainBundle] pathForResource:SPSSHConfigFile ofType:@""]);
+		TA(@"-F", [[NSUserDefaults standardUserDefaults] stringForKey:SPSSHConfigFile]);
```

Comment:
Here we should probably fall back on the default config file if SPSSHConfigFile isn't set or is null!

Reply:
This issue is fixed with a commit yesterday. It will now fallback to the Sequel Ace default file. With the implementation of the popup button, I also created a way to revert back from a user config to the one from Sequel Ace.